test_scripts/API/SetAudioStreamingIndicator/060_ATF_N_TC_SetAudioStreamingIndicator_DISALLOWED_media_app_null_permission.lua

```diff
@@ -0,0 +1,137 @@
+-- Send SetAudioStreamingIndicator(audioStreamingIndicator = "PAUSE")
+--
+-- Expected result:
+-- SDL->mobile: SetAudioStreamingIndicator_response("DISALLOWED", success:false)
+---------------------------------------------------------------------------------------------
+
+--[[ General configuration parameters ]]
+config.deviceMAC = "12ca17b49af2289436f303e0166030a21e525d266e209267433801a8fd4071a0"
+config.application1.registerAppInterfaceParams.appHMIType = {"MEDIA"}
+config.application1.registerAppInterfaceParams.isMediaApplication = true
+
+--[[ Required Shared libraries ]]
+local commonFunctions = require ('user_modules/shared_testcases/commonFunctions')
+local commonSteps = require('user_modules/shared_testcases/commonSteps')
+local commonPreconditions = require('user_modules/shared_testcases/commonPreconditions')
+local testCasesForPolicyTable = require('user_modules/shared_testcases/testCasesForPolicyTable')
+local json = require('json')
+testCasesForPolicyTable:precondition_updatePolicy_AllowFunctionInHmiLeves({"NONE", "LIMITED", "BACKGROUND", "FULL"}, "SetAudioStreamingIndicator")
+
+--[[ Local Functions ]]
+
+--[[@create_ptu_SetAudioStreamingIndicator_group: creates PTU files
+! SetAudioStreamingIndicator_group1.json: assign application with null permissions.
+! @parameters: NO
+]]
+local function create_ptu_SetAudioStreamingIndicator_group()
+  os.execute(" cp " .. config.pathToSDL .. "sdl_preloaded_pt.json" .. " " .. "files/SetAudioStreamingIndicator_group1.json" )
+  local pathToFile = config.pathToSDL .. 'sdl_preloaded_pt.json'
+  local file = io.open(pathToFile, "r")
+  local json_data = file:read("*all")
+  file:close()
+
+  local data = json.decode(json_data)
+  if(data.policy_table.functional_groupings["DataConsent-2"]) then
+    data.policy_table.functional_groupings["DataConsent-2"].rpcs = {json.null}
+  end
+
+  data.policy_table.module_config.preloaded_pt = nil
+  data.policy_table.module_config.preloaded_date = nil
+  --TODO(istoimenova): Update when problem with saving in file: "0000001": null using json.null.
+  -- data.policy_table.app_policies[config.application1.registerAppInterfaceParams.appID] = json.null
+  data.policy_table.app_policies[config.application1.registerAppInterfaceParams.appID] = null
+
+  data = json.encode(data)
+  file = io.open("files/SetAudioStreamingIndicator_group1.json", "w")
+  file:write(data)
+  file:close()
+end
+
+--[[ General Precondition before ATF start ]]
+create_ptu_SetAudioStreamingIndicator_group()
+commonSteps:DeleteLogsFiles()
+
+--[[ General Settings for configuration ]]
+Test = require('connecttest')
+require('user_modules/AppTypes')
+
+--[[ Preconditions ]]
+commonFunctions:newTestCasesGroup("Preconditions")
+
+function Test:Precondition_ActivateApp()
+  commonSteps:ActivateAppInSpecificLevel(self, self.applications[config.application1.registerAppInterfaceParams.appName])
+  EXPECT_NOTIFICATION("OnHMIStatus", {systemContext = "MAIN", hmiLevel = "FULL"})
+end
+
```

Comment:
@istoimenova, Before assigning 'null' policies it would be better to verify if "SetAudioStreamingIndicator" RPC processed successfully